### PR TITLE
Fix Servlet 5 Schema redirects

### LINKED ISSUE #5752: Fix Servlet 5 Schema redirects
Jetty-11

### PR DESCRIPTION
**Pull request by janbartel**
Closes #5752

### CODE REVIEW

File: jetty-quickstart/src/main/java/org/eclipse/jetty/quickstart/QuickStartGeneratorConfiguration.java [hunk 1]
```diff
@@ -154,9 +154,13 @@ public void generateQuickStartWebXml(WebAppContext context, OutputStream stream)
         Map<String, String> webappAttr = new HashMap<>();
         int major = context.getServletContext().getEffectiveMajorVersion();
         int minor = context.getServletContext().getEffectiveMinorVersion();
-        webappAttr.put("xmlns", "http://xmlns.jcp.org/xml/ns/javaee");
+        String ns = "https://jakarta.ee/xml/ns/jakartaee";
+        if (major < 5)
+            ns = "http://xmlns.jcp.org/xml/ns/javaee";
```
Comment (gregw) [inline jetty-quickstart/src/main/java/org/eclipse/jetty/quickstart/QuickStartGeneratorConfiguration.java]: This is faster and clearer
```suggestion
        String ns = (major < 5) ? "http://xmlns.jcp.org/xml/ns/javaee" : "https://jakarta.ee/xml/ns/jakartaee";
```

File: jetty-quickstart/src/main/java/org/eclipse/jetty/quickstart/QuickStartGeneratorConfiguration.java [hunk 2]
```diff
@@ -154,9 +154,13 @@ public void generateQuickStartWebXml(WebAppContext context, OutputStream stream)
...
+
+        webappAttr.put("xmlns", ns);
         webappAttr.put("xmlns:xsi", "http://www.w3.org/2001/XMLSchema-instance");
-        webappAttr.put("xsi:schemaLocation", "http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_" + (major + "_" + minor) + ".xsd");
+        webappAttr.put("xsi:schemaLocation", ns + " " + ns + "/web-app_" + (major + "_" + minor) + ".xsd");
```
Comment (gregw) [inline jetty-quickstart/src/main/java/org/eclipse/jetty/quickstart/QuickStartGeneratorConfiguration.java]:
```suggestion
        webappAttr.put("xsi:schemaLocation", String.format("%s %s/web-app_%d_%d.xsd", ns, ns, major, minor));
```

**Review by gregw — APPROVED** [review]
Other than some minor niggles - LGTM

**Review by joakime — APPROVED** [review]
LGTM.

**Pull request merged**